CodeEdit/Info.plist

```diff
@@ -1240,6 +1240,10 @@
 		</dict>
 	</array>
 	<key>GitHash</key>
-	<string>515c9ebc3ea3e756957a2ca8e199c41888ff233e</string>
+	<string>908830778ca1940e31e4256692adab7c8c77119b</string>
+	<key>SUFeedURL</key>
+	<string>https://CodeEditApp.github.io/CodeEdit/appcast.xml</string>
+	<key>SUPublicEDKey</key>
+	<string>/vAnxnK9wj4IqnUt6wS9EN3Ug69zHb+S/Pb9CyZuwa0=</string>
```

Comment:
I think we need `SUEnableInstallerLauncherService` set to true here since CodeEdit is sandboxed and will need that XPC service enabled.

- [Sandboxing](https://sparkle-project.org/documentation/sandboxing/)
- [`SUEnableInstallerLauncherService`](https://sparkle-project.org/documentation/customization#sandboxing-settings)

Reply:
Added, thanks for the heads up!